packages/react-core/src/components/Switch/Switch.tsx

```diff
@@ -94,7 +94,9 @@ export class Switch extends React.Component<SwitchProps & OUIAProps, { ouiaState
           {...([true, false].includes(defaultChecked) && { defaultChecked })}
           {...(![true, false].includes(defaultChecked) && { checked: isChecked })}
           disabled={isDisabled}
-          aria-labelledby={isAriaLabelledBy ? `${this.id}-on` : null}
+          aria-labelledby={
+            !isAriaLabelledBy ? null : `${this.id}-${isChecked === true && defaultChecked !== false ? 'on' : 'off'}`
```

Comment:
This may seem a little confusing... but docs say never to use both `isChecked` and `defaultChecked` together. 

`isChecked` is default `true`, so when using `defaultChecked` it's just checking that it isn't `false` to see whether or not we should use "on" or "off"

`defaultChecked` does not have a default checked, so `!== false` would always be true and now we are just checking that `isChecked === true`